flight/drone/drone.py

```diff
@@ -114,7 +114,7 @@ def send_velocity(self, north, east, down):
         msg = self._make_velocity_message(north, east, down)
         self.send_mavlink(msg)
 
-    def arm(self, mode=c.Modes.GUIDED.value):
+    def arm(self, mode=c.Modes.GUIDED):
```

Comment:
I'm anticipating all the removals of `.value` is going cause the code to not work on my machine. Your going to have to tell me (and perhaps add to the README) how I should start the flight code. Something to do with pipenv, right?

Reply:
Will add to `README.md`, the commands are as follows:
```bash
pip install pipenv --user
pipenv install --skip-lock
pipenv shell
```
then run as you will